end_to_end/fraud_detection/pipeline-e2e.ipynb

```diff
@@ -4,32 +4,37 @@
    "cell_type": "markdown",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #21.    prefix = 'fraud-detect-demo'
</pre><p>Nit: need to run black-nb? Single quotes</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3390/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
<p>Fixed</p>

Reply:
<p>Ran black-nb too as well</p>